frontend/src/AddPlaylistForm.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from "react";
+import { Link } from "react-router-dom";
+import Input from "./components/Input";
+import InputLabel from "./components/InputLabel";
+import Button from "./components/Button";
+import { Form, useForm } from "react-hook-form";
+
+const AddPlaylistForm: FC = () => {
+  const { control, register } = useForm({});
+
+  return (
+    <div className="m-2">
+      <Form
+        method="post"
+        action={`http://localhost:5000/create-playlist`}
```

Comment:
I suspect you'll appreciate pulling out the base address here at some point, since eventually we'll certainly want this to be varied

I tend to use `environment variables` even though it's a dishonest description - unlike the usual pattern, I actually _do_ want the `.env` values copied into the built static files, so I need to rebuild the code for each environment uniquely